controllers/controllers.go

```diff
@@ -382,7 +382,7 @@ func (ctrl *OAuthController) CreateClientHandler(w http.ResponseWriter, r *http.
 func (ctrl *OAuthController) PreRedirectErrorHandler(w http.ResponseWriter, r *server.AuthorizeRequest, err error) error {
 	logrus.WithField("Authorize request", r).Error(err)
 	sentry.CaptureException(err)
-	return nil
+	return err
```

Comment:
could error contain sensitive/internal data that is returned to the client?

Reply:
don't think so